imod/msw/infiltration.py

```diff
@@ -54,6 +58,21 @@ class Infiltration(MetaSwapPackage):
     )
     _to_fill = ()
 
+    _regrid_method = {
+        "infiltration_capacity": (RegridderType.OVERLAP, "mean"),
+        "downward_resistance": (RegridderType.OVERLAP, "mean"),
+        "upward_resistance": (
+            RegridderType.OVERLAP,
+            "mean",
+        ),
```

Comment:
I have introduced the same method for all parameters, and left the fine-tuning of choosing the correct parameter for each array for a future issue. If these are permeability-like parameters then it indeed be harmonic as it deals with parallel resistors. But since I couldn't find how these inputs are used by metaswap,, I wouldn't know how to upscale them. Besides @HendrikKok , also this "Paul" person could be  consulted for this I guess.

Reply:
Could you open an issue that we still need to select appropriate methods?